src/vue/forms/_issuance-form.scss

```diff
@@ -25,14 +25,11 @@
 .issuance-form__amount-label {
   margin-left: 1rem;
   padding-top: 1.8rem;
-
-  span {
-    font-size: 1.8rem;
-  }
+  font-size: 1.8rem;
 }
 
-.issuance-form__available {
-  color: #837fa1;
+.issuance-form__available-amount-hint {
+  color: $_lightPurple;
```

Comment:
do not use `$_` vars outside of `_variables.scss`

Reply:
Resolved, fixed